### DISCUSSION TITLE
Can someone tell me where the variable files related to CrewManager are?

### DISCUSSION THREAD

**Discussion by BlueSeaAndDragon** [FakeFishGames/Barotrauma#12654] [Code questions and modding support]
Can someone tell me where the variable files related to CrewManager are? I used Notepad++to open all the files in the game folder that can be read but couldn't find them...

**Comment by SomeRandomNoobKekeke** [comment]
it's in the source code [here](https://github.com/Regalis11/Barotrauma/blob/0294a78afe66ddc22062e14123aa8d083f6215af/Barotrauma/BarotraumaShared/SharedSource/GameSession/CrewManager.cs#L21) or [here](https://evilfactory.github.io/LuaCsForBarotrauma/lua-docs/code/game.gamesession/#CrewManager)

**Reply by BlueSeaAndDragon** [reply]
thank
